### PR TITLE
Add spend xch and fix spend server coins

### PR DESCRIPTION
**Pull request by Rigidity**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -310,15 +310,15 @@ if (!nativeBinding) {
...
+const { newLineageProof, newEveProof, Peer, selectCoins, sendXch, morphLauncherId, createServerCoin, mintStore, oracleSpend, addFee, masterPublicKeyToWalletSyntheticKey, masterPublicKeyToFirstPuzzleHash, masterSecretKeyToWalletSyntheticSecretKey, secretKeyToPublicKey, puzzleHashToAddress, addressToPuzzleHash, adminDelegatedPuzzleFromKey, writerDelegatedPuzzleFromKey, oracleDelegatedPuzzle, signCoinSpends, getCoinId, updateStoreMetadata, updateStoreOwnership, meltStore, signMessage, verifySignedMessage, syntheticKeyToPuzzleHash, getCost } = nativeBinding
 
 module.exports.newLineageProof = newLineageProof
 module.exports.newEveProof = newEveProof
 module.exports.Peer = Peer
 module.exports.selectCoins = selectCoins
+module.exports.sendXch = sendXch
 module.exports.morphLauncherId = morphLauncherId
 module.exports.createServerCoin = createServerCoin
-module.exports.lookupAndSpendServerCoins = lookupAndSpendServerCoins
```
Comment (Yakuhito) [inline index.js:321]: you forgot to add this

**Review by Yakuhito — APPROVED** [review]
Lookin' good

File: index.js (hunk 1 above)
Comment (Rigidity) [reply index.js:321]: wdym?

Comment (Yakuhito) [reply index.js:321]: my bad

**Review by MichaelTaylor3D — APPROVED** [review]

**Pull request merged**